.changeset/wet-mice-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@lens-protocol/wagmi": major
```

Comment:
@cesarenaldi Are the versions of `react` and `wagmi` still linked? Given it's a major change it would upgrade the number to `2.x.x` and don't know if we want to also upgrade `react`/`react-web` to `2.x.x`.